nixos/modules/services/web-servers/bluemap.nix

```diff
@@ -293,8 +297,8 @@ in {
       "${cfg.host}" = {
         root = config.services.bluemap.webRoot;
         locations = {
-          "~* ^/maps/[^/]*/tiles/[^/]*.json$".extraConfig = ''
-            error_page 404 =200 /assets/emptyTile.json;
+          "~* ^/maps/[^/]*/tiles/[^/]*.prbm$".extraConfig = ''
+            error_page 404 =200 /assets/emptyTile.prbm;
```

Comment:
The docs say these should also just return a 204 (instead of a special emtpy tile) https://bluemap.bluecolored.de/wiki/webserver/ExternalWebserversFile.html

So we only need

```
"~* ^/maps/[^/]*/tiles/".extraConfig = ''
  error_page 404 = @empty;
'';
"@empty".return = "204";
```

Doing 204 directly doesn't seem to work, as it then redirects to a path named 204, so this "@" named location thing needs to be used .